authors/filip-srdoc/15-porezi/hr_HR/final.md

```diff
@@ -0,0 +1,107 @@
+
+Jesu li onda kriptovalute dobro tj. _asset_? 
+
+IRS je u SAD-u kriptovalute [deklarirao][link] kao kapitalno dobro. Oni primjerice bitcoin smatraju "konvertibilnom" valutom koja zadovoljava sve tri funkcije novca. No, bitcoin se ne oporezuje kao strana valuta jer on nigdje nema status pravnog sredstva plaćanja iako se u nekim okruženjima njima raspolaže kao "pravom" valutom. 
+
+Australija kriptovalute smatra [sredstvom za razmjenu][link2], a ne stranom valutom jer je pojam "valuta" već definiran u njihovom zakonu s ciljem određivanja pojma “strane valute” u odnosu na domaću valutu.
+
+[Singapur][link3] virtualne valute smatra nematerijalnim oblicima pohrane vrijednosti koji se mogu mijenjati za dobra i usluge tamo gdje su prihvaćeni. Oni kažu kako primjerice bitcoin nije vrijednosni papir niti zakonito platežno sredstvo da bi ga oni regulirali, ali će zato regulirati mjenjačnice kriptovalutama da bi spriječili nezakonite radnje poput pranja novca te da će se transakcije virtualnim valutama smatrati uslugama koje podliježu općem porezu na promet.
+
+[Njemačka][link4] ne smatra virtualne valute ni stranim valutama ni elektroničkim novcem. [Austrija][link5] kaže da one nisu financijski instrument. [Slovenija][link6] kaže da nisu novčano sredstvo. Za [Ujedinjeno Kraljevstvo][link7] i [Nizozemsku][link8] novac nije ništa što nije centralizirano i zakonito sredstvo plaćanja. [Estonija, Finska i Norveška][link9] se slažu s time pa deklariraju kriptovalute kao _asset_. 
+
+Kao što vidite iz priloženog, nitko se ne može odlučiti što su kriptovalute po pravnoj prirodi, ali definitivno više država kriptovalute smatra dobrom nego financijskim instrumentom.
+
+## Oporezivanje transakcija kriptovaluta neizravnim porezima 
+
+Porezno-pravna regulacija prilično se razlikuje unutar i izvan Europske unije. Izvan EU većina država ima neku vrstu općeg poreza na promet koji se ne odnosi na plaćanja dobra valutom koja je zakonsko sredstvo plaćanja u toj zemlji. U većini zemalja izvan Europske unije postoji tendencija oslobođenja oporezivanja kriptovaluta neposrednim porezima jer države nisu spremne prihvatiti pravnu kvalifikaciju virtualnih valuta kao pravog novca, nego ga one ih one gledaju kao _assets_ odnosno dobro ili imovinu. Pojednostavljeno, to znači da ako ste kriptovalutu koja se smatra _assetom_ kupili "pravom" valutom onda na to ne plaćate porez. 
+
+Unutar Europske unije zakoni su ipak malo drugačiji.
+Izravni porezi su u nadležnosti države članice, no neizravni porezi su u nadležnosti Europske unije koja je željela stvoriti jedinstveno tržište. Glavni europski porez je porez na dodanu vrijednost (PDV). U zahtjevu zvanom Hedqvist kojeg je Švedska mjenjačnica poslala Europskoj uniji 2014. godine traži se očitovanje EU o oporezivanju kriptovaluta odnosno o plaćanju PDV-a na zamjene tradicionalnih valuta za virtualne valute. S obzirom da je kupnja i prodaja kriptovaluta za fiat valute usluga, ona bi trebala podlijegati PDV-u. No, sud EU [kaže][pu]: 
```

Comment:
Link na hedqvist ili tu mjenjačnicu, prvi put čujem

Reply:
nema linka, mogu eventualno negdje nac taj dogadaj na netu al oni nisu nista naveli

Reply:
http://www.nortonrosefulbright.com/knowledge/publications/133585/bitcoins-ecj-rules-that-buying-and-selling-bitcoin-is-exempt-from-vat